spec/system/bus_stops/managing_bus_stops_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+
+describe 'managing stops as an admin' do
+  let(:admin) { create :user, :admin }
+  let!(:bus_stop) { create :bus_stop }
+  before :each do
+    when_current_user_is admin
+    visit manage_bus_stops_url
+  end
+  context 'delete button' do
+    it 'deletes the specific bus stop' do
+      expect(page).to have_selector 'table.manage tbody tr', count: 1
```

Comment:
What if there were more than one `bus_stop` on this page? Would it delete the right objet?

Reply:
As long as the _objet_ doesn't have the same name as another.

Reply:
(Bien que j'aimerais croire que @jforgue essayait de parler français, je crois qu'il a fait une erreur typographique.)

Reply:
@jforgue you spelled my name wrong. And I agree, I shouldn't rely on names for this test.